integrations/shuttle-axum/src/lib.rs

```diff
@@ -0,0 +1,41 @@
+//! Shuttle service integration for the Axum web framework.
+//! ## Example
+//! ```rust,no_run
+//! use shuttle_axum::AxumService;
+//!
+//! async fn hello_world() -> &'static str {
+//!     "Hello, world!"
+//! }
+//!
+//! #[shuttle_axum::main]
+//! async fn axum() -> shuttle_service::ShuttleAxum {
```

Comment:
```suggestion
//! async fn axum() -> impl shuttle_service::Service {
```
Not sure if we can do this?

Reply:
As discussed probably not, but I'll leave this open in case we have any bright ideas.